From b8b0b5fc79a2c5f92db015caf2b69c3db2a42e55 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Mon, 8 Apr 2019 14:21:55 +0800
Subject: [PATCH 09/30] drm/imx/hdp: fix
 drm_hdmi_vendor_infoframe_from_display_mode compile error

In upstream commit f1781e9bb2dd ("drm/edid: Allow HDMI infoframe without
VIC or S3D"), drm_hdmi_vendor_infoframe_from_display_mode increases an
argument "struct drm_connector *connector", so add it to imx-hdmi.c

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdmi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdmi.c b/drivers/gpu/drm/imx/hdp/imx-hdmi.c
index a3d53bf424da..303a6af6dc93 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdmi.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdmi.c
@@ -110,7 +110,7 @@ static int hdmi_vendor_info_set(struct imx_hdp *hdp,
 	int ret;
 
 	/* Initialise vendor frame from DRM mode */
-	ret = drm_hdmi_vendor_infoframe_from_display_mode(&frame, mode);
+	ret = drm_hdmi_vendor_infoframe_from_display_mode(&frame, &hdp->connector, mode);
 	if (ret < 0) {
 		DRM_DEBUG("Unable to init vendor infoframe: %d\n", ret);
 		return -1;
-- 
2.17.1

